superbench/runner/runner.py

```diff
@@ -438,12 +452,17 @@ def _run_proc(self, benchmark_name, mode, vars):
             # we do not expect timeout in ansible unless subprocess hangs
             ansible_runner_config['timeout'] = timeout + 60
 
-        rc = self._ansible_client.run(ansible_runner_config, sudo=(not self._docker_config.skip))
+        # overwrite ansible runner's default signal handler with main process's
```

Comment:
Can't overwrite ansible runner's default signal handler using your handler?

Reply:
ansible is running in child process, current handler is registered in main process and traps signals in both ansible runner and other python logic